src/app/home-page/widget/starred-box/starred-box.component.html

```diff
@@ -1,192 +1,204 @@
 <mat-card class="h-100">
-  <div id="header-bar" fxLayout="row" fxLayoutAlign="space-between">
-    <mat-card-header fxLayout="row">
-      <mat-card-title>Starred</mat-card-title>
-    </mat-card-header>
-    <div id="total-starred" fxLayout="row" fxLayoutGap="1rem" class="size-small pb-3">
-      <div [ngPlural]="totalStarredWorkflows" id="starred-workflows" fxLayout="row" fxLayoutAlign="start center">
-        <ng-template ngPluralCase="=0"> <img src="../assets/svg/star-off.svg" alt="star" class="mr-1" />0 Workflows</ng-template>
-        <ng-template ngPluralCase="=1"> <img src="../assets/svg/star-on.svg" alt="star" class="mr-1" />1 Workflow</ng-template>
```

Comment:
Are these images used anymore since you're using the mat-icons now? If not, I think you can delete them